README.md

```diff
@@ -213,6 +215,9 @@ clasp <command>
 
 #### Run Tests (experimental)
 
+Check out `/tests/test.ts` and change `describe.skip(...)` to `describe(...)` for relevant tests.
```

Comment:
Is there a better way to run individual tests?

Reply:
Don't say `Check out /tests/test.ts\`. It's wordy. Accommodate for most people reading this.

I'm guessing if someone searches for this section
90% of people want to run tests.
10% of people want to add a test.

---

So just state how to run all tests. 1 sentence.
If you find a way to run a single test, you can state that too in 1 sentence/command.